portafly/src/i18n/locales/en/account_settings/personal/personal_details.yml

```diff
@@ -1,5 +1,63 @@
+page_title_description: Page title of the subsection for Personal details.
+
+# Top area
+
 body_title: Personal details
-body_title_description: Body title of the page for Personal details
+body_title_description: Body title of the page for Personal details.
+
+# Main area
+
+# Tabs
+
+user_data_title: User data
+user_data_description: This is the title of the tab containing the user's personal information.
+change_password_title: Change password
+change_password_description: This is the title of the tab containing the user's current password field and new password field.
+
+# User data form
+
+username_label: Username
+username_label_description: Label for the field that holds the current username.
+username_validation: "{{username}} is too short. Use a minimum of 3 characters. Do not use spaces. You can include letters, numbers, and the following special characters . - _ @ +"
+username_validation_description: Input field that holds the current username, and validates the content.
+
+email_label: Email
+email_label_description: Label for the field that holds the valid user's email address.
+email_validation: "Use a valid email such as <i>admin@domain.com</i>. The email can contain both lower and upper case letters, numbers, and the following characters . - _ @ +"
+email_validation_description: Input field that holds the valid user's email address.
+
+first_name_label: First name
+first_name_label_description: Label for the field that holds the user's first name.
+
+last_name_label: Last name
+last_name_label_description: Label for the field that holds the user's last name.
+update_details_button: Update details
+update_details_description: The user clicks this button to procede with updating the edited personal information.
+
+# Modal: Update details
+
+current_password_title: Provide your current password to update your personal details.
+current_password_title_description: Body title of modal dialog which prompts the user to provide their current password to make the update.
+current_password_label: Current password
+current_password_label_description: Label for the field for the user to enter a new password.
+current_password_helper: When you update your personal details, all other active sessions for this user will end. This happens in order to confirm that it is you, and not somebody else trying to lock you out of your account.
+current_password_helper_description: Text to let the user know when they updates their personal details, all other active session are ended for security reasons.
+#-------# Confirm button: available in shared.yml
+#-------# Cancel button: available in shared.yml
+
+# Change password tab
+
+password_update_options:
+ success: You have successfully updated your user details
+ error: Your user details were not updated. Please review the password you provided
+current_password_validation_description: Input field for the user to enter the current password.
```

Comment:
Have you removed the string for `current_password_validation`?
The description is here, but not the string.

Reply:
@dfennessy Looking at the context, the description and the string name should be connected:

```
password_update_options:
 success: You have successfully updated your user details
 error: Your user details were not updated. Please review the password you provided
password_update_options_description: Input field for the user to enter the current password.
```